Description: ubuntu 16 ppc build fixes
Author: Jethro Grassie <jtg@xtrabass.com>
Origin: upstream, https://github.com/monero/monero-core/commit/
Last-Update: 2018-11-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -148,7 +148,7 @@
   set(PPC     0)
 endif()
 
-if(ARCH_ID STREQUAL "powerpc")
+if(ARCH_ID STREQUAL "powerpc" OR ARCH_ID STREQUAL "ppc")
   set(PPC64LE 0)
   set(PPC64   0)
   set(PPC     1)
